Notes:
All my configuration is in this file

Configuration of the following patches:
 * highlight
--- a/config.def.h
+++ b/config.def.h
@@ -4,14 +4,18 @@
 static int topbar = 1;                      /* -b  option; if 0, dmenu appears at bottom     */
 /* -fn option overrides fonts[0]; default X11 font or font set */
 static const char *fonts[] = {
-	"monospace:size=10"
+	"Source Code Pro:style=Medium:size=11"
 };
 static const char *prompt      = NULL;      /* -p  option; prompt to the left of input field */
 static const char *colors[SchemeLast][2] = {
-	/*     fg         bg       */
-	[SchemeNorm] = { "#bbbbbb", "#222222" },
-	[SchemeSel] = { "#eeeeee", "#005577" },
-	[SchemeOut] = { "#000000", "#00ffff" },
+	/*                            fg         bg        */
+	[SchemeNorm] =              { "#6272a4", "#282a36" },
+	[SchemeSel] =               { "#f8f8f2", "#44475a" },
+	[SchemeSelHighlight] =      { "#51afef", "#44475a" },
+	[SchemeNormHighlight] =     { "#51afef", "#282a36" },
+	[SchemeOut] =               { "#000000", "#00ffff" },
+	[SchemeOutHighlight] =      { "#ffc978", "#00ffff" },
+	[SchemeHp] =                { "#ff5555", "#282a36" },
 };
 /* -l option; if nonzero, dmenu uses vertical list with given number of lines */
 static unsigned int lines      = 0;
